crates/primitives/src/signature/sig.rs

```diff
@@ -314,6 +314,11 @@ impl Signature {
         sig
     }
 
+    /// Turn this signature into its hex-encoded representation.
+    pub fn as_hex_bytes(&self) -> Bytes {
```

Comment:
this too, 1 the name is wrong 2 just use as_bytes, if you need it as Bytes then do .into or Bytes::from